src/Domain/Entities/UserCreditCard.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace CloudDrive.Domain.Entities
+{
+    public class UserCreditCard
+    {
+        public int Id { get; set; }
+        public int? UserId { get; set; }
+        public string CreditCardNumber { get; set; }
+        public string CreditCardSecretCode { get; set; } //CVV
+        public string HolderName { get; set; }
+        public string ExpireMonth { get; set; } // the month should be like 08, 06 
```

Comment:
Please convert it into an int, as each part is still a number not a string, we dont care about leading 0